gui/build.rs

```diff
@@ -8,6 +9,23 @@ const LINUX_INCLUDE: &str = r#"
 "#;
 
 fn main() {
+    if std::env::var("CARGO_FEATURE_GUI_SLINT").is_ok_and(|var| var == "1") {
+        build_bin();
+    } else if std::env::var("CARGO_FEATURE_QT_FFI").is_ok_and(|var| var == "1") {
```

Comment:
This should not be `else if`. See https://doc.rust-lang.org/cargo/reference/features.html#feature-unification for more information about this.